modules/nf-core/biscuit/align/main.nf

```diff
@@ -2,18 +2,19 @@ process BISCUIT_ALIGN {
     tag "$meta.id"
     label 'process_high'
 
-    conda "bioconda::biscuit=1.0.2.20220113 bioconda::samtools=1.15=h1170115_1"
+    conda (params.enable_conda ? "bioconda::biscuit=1.1.0.20220707 bioconda::samtools=1.16.1" : null)
```

Comment:
Can you revert the Conda declaration here and in the other files ? #2654 was merged earlier this week and moved away from having to use `params.enable_conda`